desktop/src/main/java/bisq/desktop/main/support/dispute/agent/SignedOfferView.java

```diff
@@ -0,0 +1,425 @@
+
+    private SignedOffer selectedSignedOffer;
+
+    private XmrWalletService xmrWalletService;
+
+    private TradeManager tradeManager;
+
+    private ContextMenu contextMenu;
+
+    private OfferDetailsWindow offerDetailsWindow;
+
+    private CoreApi coreApi;
+
+    @Inject
+    public SignedOfferView(OpenOfferManager openOfferManager, XmrWalletService xmrWalletService,
+                           TradeManager tradeManager, OfferDetailsWindow offerDetailsWindow,
+                           CoreApi coreApi) {
+        this.openOfferManager = openOfferManager;
+        this.xmrWalletService = xmrWalletService;
+        this.tradeManager = tradeManager;
+        this.offerDetailsWindow = offerDetailsWindow;
+        this.coreApi = coreApi;
+    }
+
+    private Offer fetchOffer(SignedOffer signedOffer) {
+        Trade trade = tradeManager.getTrade(signedOffer.getOfferId());
+
+        return trade != null ? trade.getOffer() : null;
+    }
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Life cycle
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void initialize() {
+        Label label = new AutoTooltipLabel(Res.get("support.filter"));
+        HBox.setMargin(label, new Insets(5, 0, 0, 0));
+        HBox.setHgrow(label, Priority.NEVER);
+
+        filterTextField = new InputTextField();
+        Tooltip tooltip = new Tooltip();
+        tooltip.setShowDelay(Duration.millis(100));
+        tooltip.setShowDuration(Duration.seconds(10));
+        filterTextField.setTooltip(tooltip);
+        HBox.setHgrow(filterTextField, Priority.NEVER);
+
+        filterTextField.setText("open");
+
+        setupTable();
+    }
+    @Override
+    protected void activate() {
+        super.activate();
+
+        ObservableList<SignedOffer> observableList = FXCollections.observableArrayList();
+        observableList.addAll(openOfferManager.getSignedOffers());
+
+        SortedList<SignedOffer> sortedList = new SortedList<>(observableList);
+        sortedList.comparatorProperty().bind(tableView.comparatorProperty());
+        sortedList.addListener((ListChangeListener<SignedOffer>) c -> tableView.refresh());
+        tableView.setItems(sortedList);
+
+        contextMenu = new ContextMenu();
+        MenuItem item1 = new MenuItem(Res.get("support.contextmenu.penalize.msg",
+                Res.get("shared.maker")));
+        contextMenu.getItems().addAll(item1);
+
+        tableView.setRowFactory(tv -> {
+            TableRow<SignedOffer> row = new TableRow<>();
+            row.setOnContextMenuRequested(event -> {
+                contextMenu.show(row, event.getScreenX(), event.getScreenY());
+            });
+            return row;
+        });
+
+        item1.setOnAction(event -> {
+            selectedSignedOffer = tableView.getSelectionModel().getSelectedItem();
+            Offer offer = fetchOffer(selectedSignedOffer);
```

Comment:
Remove offer fetch.

Reply:
That line of code no longer exists in the current working code.

What you have is below:

![Screenshot from 2023-02-28 16-14-56](https://user-images.githubusercontent.com/20237127/221896407-e1498975-a246-4448-958a-7d94af70b0ed.png)

Offer object cannot be null (i.e offer == null is tested).